sample-apps/gradle/build.gradle

```diff
@@ -0,0 +1,35 @@
+group 'samples'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+apply plugin: 'application'
+apply plugin: "com.github.johnrengelman.shadow"
+
+sourceCompatibility = 1.8
+
+mainClassName = "samples.App"
+
+buildscript {
+
+    repositories {
+        mavenCentral()
+//        maven { url 'http://maven.aliyun.com/nexus/content/groups/public/' }
```

Comment:
This sample will be used in many projects, so keep it as clean as possible. Delete this.